app/views/search/_results_list.mustache

```diff
@@ -29,25 +29,40 @@
     {{/results}}
   </ol>
 
-  <ul class="previous-next-navigation">
-    {{#has_previous_page?}}
-      <li class="previous">
-        <a href="{{previous_page_link}}">
-          Previous <span class="visuallyhidden">page</span>
-          <span class="page-numbers">{{previous_page_label}}</span>
-        </a>
-      </li>
-    {{/has_previous_page?}}
+  <!-- TODO: replave this with the static helper -->
+  <nav class="govuk-previous-and-next-navigation" role="navigation" aria-label="Pagination">
```

Comment:
Where did you pull this markup from?

It's an old static version. New one here:
https://govuk-static.herokuapp.com/component-guide/previous_and_next_navigation

The version you've added doesn't work with Static master:
![screen shot 2017-10-23 at 13 07 31](https://user-images.githubusercontent.com/319055/31888249-2f4490d0-b7f3-11e7-95f2-71712d180b54.png)

We avoid doing this as it means the static component is no longer isolated – its styles and template can't be updated in one place, and any future iteration would break here.

There's some steps needed to switch from mustache to the component, probably following the same approach that finder-frontend uses in finders to dynamically update.

Maybe it's best to leave obviously broken for now?

Reply:
@fofr have reverted the pagination changes, let pencil in to fix them when we look at the whitespace issue.